src/scc.ts

```diff
@@ -283,18 +213,12 @@ const completionSpec: Fig.Spec = {
       description: "Limit to these file extensions (comma-separated list)",
       args: {
         name: "strings",
-        generators: {
-          getQueryTerm: ",",
-          script: "scc --languages",
-          postProcess: (out) => {
-            const { extensions } = processSccLanguages(out);
-            return Object.entries(extensions).map(([extension, language]) => ({
-              name: extension,
-              description: language,
-              icon: "fig://icon?type=string",
-            }));
-          },
-        },
+        // fun fact - keyValueList allows you to omit values, so you can
+        // use it just as a list.
+        generators: keyValueList({
+          keys: suggestExtensions,
+          cache: true,
+        }),
```

Comment:
Once https://github.com/withfig/autocomplete-tools/pull/64 lands this will become a `valueList`
```suggestion
        generators: valueList({
          values: suggestExtensions,
          cache: true,
          insertDelimiter: true,
        }),
```

Reply:
Do you prefer to wait for that pr?

Reply:
It's actually done right now if you want to review that and publish!